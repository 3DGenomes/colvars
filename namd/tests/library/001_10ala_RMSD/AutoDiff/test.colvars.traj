# step         rmsdA                 fs_rmsdA               rmsdB                 fs_rmsdB               rmsdC                 fs_rmsdC               rmsdD                 fs_rmsdD               rmsdE                 fs_rmsdE              
           0    2.61009345658189e-01  0.00000000000000e+00   1.27145912684355e+01  0.00000000000000e+00   6.68502465870546e-01  0.00000000000000e+00   9.31703240469482e-01  0.00000000000000e+00   1.24224459145532e+00  0.00000000000000e+00 
