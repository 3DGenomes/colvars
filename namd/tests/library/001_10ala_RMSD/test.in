colvarsTrajFrequency 1


colvar {
    name rmsdA

    outputSystemForce on
    #  outputAppliedForce on
    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdB

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            rotateReference
            centerReference
            refpositionsfile ../Common/pi-ideal.xyz
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdC

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            centerReference
            refpositionsfile ../Common/da.xyz
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdD

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            rotateReference
            refpositionsfile ../Common/da.xyz
        }
        refpositionsfile ../Common/da.xyz
    }
} 


colvar {
    name rmsdE

    outputSystemForce on

    width 1.0 

    lowerBoundary 0.0
    upperBoundary 5.0

    lowerWallConstant 100.0
    upperWallConstant 100.0

    rmsd {
        atoms {
            atomNumbers { 1 2 3 4 5 6 7 8 }
            centerReference no
            rotateReference no
        }
        refpositionsfile ../Common/da.xyz
    }
} 


harmonic {
    colvars        rmsdA rmsdB rmsdC rmsdD rmsdE
    forceConstant  1.0
    centers        0.0   0.0   0.0   0.0   0.0   
}
