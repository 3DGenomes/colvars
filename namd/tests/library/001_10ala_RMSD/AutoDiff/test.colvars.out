colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  http://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is available.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsdA"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069, total charge = -0.09.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsdB"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = on
colvars:       # rotateReference = on
colvars:       # enableFitGradients = on [default]
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/pi-ideal.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069, total charge = -0.09.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsdC"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = on
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/da.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069, total charge = -0.09.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsdD"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = on
colvars:       # enableFitGradients = on [default]
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/da.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069, total charge = -0.09.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rmsdE"
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off
colvars:       # rotateReference = off
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 8 atoms initialized: total mass = 43.069, total charge = -0.09.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     WARNING: explicit fitting parameters were provided for atom group "atoms".
colvars:   All components initialized.
colvars:   # width = 1
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 100
colvars:   # lowerWall = 0 [default]
colvars:   # upperBoundary = 5
colvars:   # upperWallConstant = 100
colvars:   # upperWall = 5 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 5 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsdA, rmsdB, rmsdC, rmsdD, rmsdE }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 1
colvars:   # centers = { 0, 0, 0, 0, 0 }
colvars:   # targetCenters = { 0, 0, 0, 0, 0 } [default]
colvars:   # targetForceConstant = 0 [default]
colvars:   # outputCenters = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (8 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group rmsdA:0/0. 8 atoms: total mass = 43.069.
colvars: Re-initialized atom group rmsdB:0/0. 8 atoms: total mass = 43.069.
colvars: Re-initialized atom group rmsdC:0/0. 8 atoms: total mass = 43.069.
colvars: Re-initialized atom group rmsdD:0/0. 8 atoms: total mass = 43.069.
colvars: Re-initialized atom group rmsdE:0/0. 8 atoms: total mass = 43.069.
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (8 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
